tests/framework/extensions/clusters/clusters.go

```diff
@@ -225,28 +225,41 @@ func NewK3SRKE2ClusterConfig(clusterName, namespace, cni, cloudCredentialSecretN
 func HardenK3SRKE2ClusterConfig(clusterName, namespace, cni, cloudCredentialSecretName, kubernetesVersion string, machinePools []apisV1.RKEMachinePool) *apisV1.Cluster {
 	v1Cluster := NewK3SRKE2ClusterConfig(clusterName, namespace, cni, cloudCredentialSecretName, kubernetesVersion, machinePools)
 
-	v1Cluster.Spec.RKEConfig.MachineGlobalConfig.Data["kube-apiserver-arg"] = []string{
-		"enable-admission-plugins=NodeRestriction,PodSecurityPolicy,ServiceAccount",
-		"audit-policy-file=/var/lib/rancher/k3s/server/audit.yaml",
-		"audit-log-path=/var/lib/rancher/k3s/server/logs/audit.log",
-		"audit-log-maxage=30",
-		"audit-log-maxbackup=10",
-		"audit-log-maxsize=100",
-		"request-timeout=300s",
-		"service-account-lookup=true",
-	}
+	if strings.Contains(kubernetesVersion, "k3s") {
```

Comment:
Double-checking...so the default will assume RKE2, correct? In other words, if the `kubernetesVersion` does not include `k3s`, then `rke2` will be assumed as the `kubernetesVersion`, correct?

Reply:
That is correct. I tried this locally for both k3s and RKE2 cluster and it worked for the both as expected. just to be extra cautious I'll get a k3s jenkins run too so that it works as expected.